geode-docs/configuring/running/cluster-management-service.html.md.erb

```diff
@@ -22,71 +22,92 @@ limitations under the License.
 **Note:** This feature is experimental and is subject to change in future releases of <%=vars.product_name_long%>.
 
 The cluster management service enables you to change the configuration of
-the cluster (for example, create/destroy regions, indexes, or gateway receivers/senders) via rest service, and have these
-changes replicated on all the applicable servers and persisted in the cluster configuration service.
+the cluster using a REST interface,
+and have these changes replicated to all the applicable servers
+and persisted within the cluster configuration service.
+You can create and destroy regions, indexes, and gateway receivers and senders.
```

Comment:
we can only create gateway receivers, can't delete them. And we haven't done anything with senders.